synapseclient/client.py

```diff
@@ -357,7 +368,7 @@ def log_response(response: httpx.Response) -> None:
         # Check for a config file
         self.configPath = configPath
         if os.path.isfile(configPath):
-            config = self.getConfigFile(configPath)
+            config = get_config_file(configPath)
```

Comment:
If we want to keep this consistent with other function calls, do we want this to be `self.configPath`?

Reply:
There is no difference in this case as just above we're doing `self.configPath = configPath`